jitpack.yml

```diff
@@ -0,0 +1,2 @@
+install:
+   - cd android-exoplayer && ./gradlew publishToMavenLocal
```

Comment:
Does this publish a version of exoplayer to our Maven repo? And am I correct to assume we do this instead of using the upstream one because of the Gradle 7 tweaks done in this PR?

Reply:
No, it publishes to Jitpack as this is the jitpack configuration file. The way it works is they publish it to `mavenLocal` which is the `~/.m2` directory and then update a few things and serve it to the public. It's a nice hack, but a hack nevertheless. You can run `rm -rf ~/.m2 && ./gradlew publishToMavenLocal && exa ~/.m2 --tree` to see what publishing this artifact does. (If you don't have [exa](https://github.com/ogham/exa), this will be a good chance to install it 😄 )

The reason we publish `android-exoplayer` instead of `android` is that's the one that's used in mobile-gutenberg, but they want to keep the old one.